src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -1892,11 +1906,7 @@ export default class ResultsViewOncoprint extends React.Component<
                 _.indexOf(MUTATION_SPECTRUM_CATEGORIES, value)
             ];
         } else if (this.selectedClinicalTrack.datatype === 'string') {
-            if (
-                (this.selectedClinicalTrack.label === 'Sample Type' ||
-                    this.selectedClinicalTrack.label === 'Sample type id') &&
-                value === 'Mixed'
-            ) {
+            if (value === 'Mixed') {
```

Comment:
maybe put comment here expaining what "Mixed" means.  Our code isn't well commented to begin with but no time like refactoring when you understand the code to improve the comments